src/hotspot/cpu/aarch64/globals_aarch64.hpp

```diff
@@ -127,6 +127,10 @@ define_pd_global(intx, InlineSmallCode,          1000);
           "Branch Protection to use: none, standard, pac-ret")          \
   product(bool, AlwaysMergeDMB, true, DIAGNOSTIC,                       \
           "Always merge DMB instructions in code emission")             \
+  product(bool, NeoverseN1Errata1542419, false, DIAGNOSTIC,             \
+          "Enable workaround for Neoverse N1 erratum 1542419")          \
+  product(bool, UseDeferredICacheInvalidation, false, DIAGNOSTIC,       \
+          "Defer multiple ICache invalidation to single invalidation")  \
```

Comment:
Since the `ICacheInvalidationContext` is in shared code, and I suppose x86_64 would also benefit from this (at least eventually), this sounds like `globals.hpp` option.